music21/harmony.py

```diff
@@ -2499,6 +2500,57 @@ def transpose(self: NCT, _value, *, inPlace=False) -> NCT | None:
 
 # ------------------------------------------------------------------------------
 
+class FiguredBass(Harmony):
+    '''
+    The FiguredBassIndication objects store information about thorough bass figures.
+    It is created as a representation for <fb> tags in MEI and <figured-bass> tags in MusicXML.
+    The FiguredBassIndication object derives from the Harmony object and can be used
+    in the following way:
+    >>> fb = harmony.FiguredBass('#,6#', correspondingPart='P1')
+    >>> fb
+    <FiguredBass figures: #,6# correspondingPart: P1>
+
+    The single figures are stored as figuredBass.notation.Figure objects:
+    >>> fb.figNotation.figures[0]
+    <music21.figuredBass.notation.Figure 3 <Modifier # sharp>>
+    '''
+
+
+    def __init__(self,
+                 figureString: str | list[str] | None = None,
+                 correspondingPart: str | None=None,
```

Comment:
sorry, I wasn't clear about this.  Anything ending in Part (part, correspondingPart, activePart, etc.) looks like it should be a Part object.  partId would be clear that it is a string or None.

But I don't know why this would need this information at all -- the Part that an object is part of can always be retrieved by:

```
fb.getContextByClass(stream.Part)
```

then if the Figure is moved to a new Part or there are two parts with the same id, the FiguredBass will always be with the right part.  This is why Note, Clef, etc. don't need a correspondingPart object.